HISTORY.md

```diff
@@ -2,12 +2,22 @@
 
 http://visjs.org
 
-## 2019-07-19, version 6.0.0
+## 2019-10-21, version 6.0.0
```

Comment:
please make sure to update this before making the actual release!

Reply:
Of course